docs/requirements/requirements.md

```diff
@@ -0,0 +1,105 @@
+# Hardware requirements of TwPM module
+
+[tcg_client_v13r27]:<https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClientTPMInterfaceSpecification_TIS__1-3_27_03212013.pdf>
+[tcg_client_PTP_v1p5r14]:<https://trustedcomputinggroup.org/wp-content/uploads/PC-Client-Specific-Platform-TPM-Profile-for-TPM-2p0-v1p05p_r14_pub.pdf>
+
+## Requirements
+
```

Comment:
@3mdeb-karolzmyslowski Please explain to the readers what this doucument is about, and what are it's goals.

Reply:
(ack, I'm working on it).